practice-one/src/components/CustomerItem/CustomerItem.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import "./CustomerItem.css";
+import { menuDot } from "@assets/images";
+import { CustomerInfo, Gender, ContextMenu } from "@components";
+
+const CustomerItem = ({ avatar, name, email, phoneNumber, gender }) => {
+  const [isContextMenuVisible, setIsContextMenuVisible] = useState(false);
```

Comment:
rename `isVisibleContextMenu`

Reply:
I changed from isContextMenuVisible to isShow
![image](https://github.com/datdragon777/React-Training/assets/74806826/d3d32c94-7ea4-4792-ad51-7d9a1e9c6b85)